server/graphql/v2/input/PersonalTokenCreateInput.ts

```diff
@@ -0,0 +1,19 @@
+import { GraphQLInputObjectType, GraphQLList, GraphQLString } from 'graphql';
+
+import { OAuthScope } from '../enum/OAuthScope';
+
+import { AccountReferenceInput } from './AccountReferenceInput';
+
+export const PersonalTokenCreateInput = new GraphQLInputObjectType({
+  name: 'PersonalTokenCreateInput',
+  description: 'Input type for PersonalToken',
+  fields: () => ({
+    name: { type: GraphQLString },
+    scope: { type: new GraphQLList(OAuthScope) },
+    expiresAt: { type: GraphQLString },
+    account: {
+      type: AccountReferenceInput,
+      description: 'The account to use as the owner of the application. Defaults to currently logged in user.',
+    },
```

Comment:
Given that personal tokens can only be created on personal (USER) accounts - at the difference of API keys - I think we should remove this argument altogether and consider that personal tokens are always referring to the currently logged-in user. @znarf wdyt?

Reply:
Agree, that can make things simpler.

But for API consistency, maybe it's better to pass the Account. Let's say that a root admin wants to create a token for someone else, maybe the API design should allow this.

Not saying this particularly desirable here, but wondering if it's a good practice to guess an Account from the loggedInUser at any time.

Reply:
> Not saying this particularly desirable here, but wondering if it's a good practice to guess an Account from the loggedInUser at any time.

I think it's ok for things that are scoped for the logged-in user not to require you to pass any account. In all cases, we're already doing it here since we default to the logged-in user if nothing was passed for `account`.